third_party/silabs/SiWx917_sdk.gni

```diff
@@ -216,27 +195,25 @@ template("efr32_sdk") {
     cflags += [
       "-Wno-maybe-uninitialized",
       "-Wno-shadow",
+      "-Wno-empty-body",
```

Comment:
All those flag disabled to build your sdk will need to be addressed asap. Please create a task to track this.
The list is extensive and some of the flags are really important to prevent risky bugs

Reply:
Yeah, already discussing this with the sdk team. Most of them are addressed in the latest package which we will be getting this month end